### PR TITLE
Add troubleshooting topic on Yellow kit installation

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- Symptom: yellow LED keeps blinking, installation fails

### CODE REVIEW

File: src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <p>
+    <span> During installation on a Yellow Kit, the yellow LED keeps blinking in a heartbeat pattern.</span
+    >
+  </p>
+  <p><span></span></p>  
+  <p><span><h1>Cause</h1></span></p>
+  <p>
+    <span>If this LED pattern persists during the installation, and the LED never changes to solid on, this indicates an issue with the installer. It may not be able to connect to the internet.</span>
+  </p>
+  <p><span><h1>Resolution</h1></span></p>
```
Comment (agners) [inline src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html:18]:
```suggestion
  <p><span><h1>Resolution</h1></span></p>
  <p>
    <span>You can install Home Assistant OS directly onto the internal eMMC using the `rpiboot` boot method. Follow this steps to do so:</span
    >
  </p>
```

File: src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <p>
+    <span>
+      <ol>
+        <li>Download the Yellow operating system.
+          <ul><li>Go to the <a href="https://github.com/home-assistant/operating-system/releases">operating system page</a> on github and download the latest version of the `haos_yellow-xx.x.img.xz` file.
+          </li></ul></li>
+          <li>Reinstall the operating system on Yellow. <ul><li>Open the procedure on <a href="https://github.com/home-assistant/operating-system/releases">how to reinstall the operating system</a>. </li><li>Follow the procedure <i>Option 2: Reinstall Home Assistant OS using rpiboot</i>  up to step 8.</li>
```
Comment (agners) [inline src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html:25]:
```suggestion
          <li>Reinstall the operating system on Yellow. <ul><li>Open the procedure on <a href="/guides/reinstall-os/#install-os-rpiboot">how to reinstall the operating system</a>. </li><li>Follow the procedure <i>Option 2: Reinstall Home Assistant OS using rpiboot</i>  up to step 8.</li>
```

Comment (c0ffeeca7) [reply src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html:25]: thank you for the catch! :pray:

File: src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+layout: guide
+guideName: "Symptom: Installation on Yellow kit: yellow LED keeps blinking"
+---
+
+<div class="container">
+  
+  <p><span><h1>Symptom</h1></span></p>
+  <p>
+    <span> During installation on a Yellow Kit, the yellow LED keeps blinking in a heartbeat pattern.</span
```
Comment (agners) [inline src/guides/install-yellow-light-keeps-flashing-in-heartbeat.html:10]:
```suggestion
    <span> During Home Assistant OS installation on a Yellow Kit, the yellow LED keeps blinking in a heartbeat pattern.</span
```

**Review by agners — APPROVED** [review]

**Pull request merged**